sagemaker_batch_transform/tensorflow_open-images_tfrecord/tensorflow-serving-tfrecord-python-sdk.ipynb

```diff
@@ -61,7 +61,9 @@
   {
```

Comment:
<p>The downloaded binary objects might get accidently be added to git tracking history. Consider download those binaries outside a git repo. For example</p><pre class="ql-syntax" spellcheck="false">!!aws s3 cp s3://sagemaker-sample-data-{region}/batch-transform/open-images/model/resnet_v2_fp32_savedmodel_NCHW_jpg.tar.gz /tmp/resnet_v2_model

!tar -zxf /tmp/resnet_v2_model/resnet_v2_fp32_savedmodel_NCHW_jpg.tar.gz -C /tmp/resnet_v2_model
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>fixed for both notebooks</p>